workload/src/main/java/com/google/finapp/WorkloadClient.java

```diff
@@ -17,30 +17,81 @@
 import com.google.finapp.FinAppGrpc.FinAppBlockingStub;
 import com.google.protobuf.ByteString;
 import io.grpc.ManagedChannel;
+import io.grpc.Status;
 import io.grpc.StatusRuntimeException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-/** A gRPC client for the finance sample app. */
-public class WorkloadClient {
+/**
+ * A gRPC client for the finance sample app. Runs until externally terminated and performs randomly
+ * chosen queries using available methods.
+ */
+public class WorkloadClient implements Runnable {
 
   private final FinAppBlockingStub blockingStub;
   private static final Logger logger = Logger.getLogger(WorkloadClient.class.getName());
+  private final List<ByteString> ids;
+  private final Random random = new Random();
 
   private WorkloadClient(ManagedChannel channel) {
     this.blockingStub = FinAppGrpc.newBlockingStub(channel);
+    this.ids = new ArrayList<>();
   }
 
   public static WorkloadClient getWorkloadClient(ManagedChannel channel) {
     return new WorkloadClient(channel);
   }
 
-  public ByteString createAccount(
-      String balance, CreateAccountRequest.Type type, CreateAccountRequest.Status status)
+  @Override
+  public void run() {
+    for (int i = 0; i < 2; i++) { // ensure that > 2 accounts exists for future methods
+      addAccountWithRandomBalance();
+    }
+    int numMethods = 2; // must be updated when new methods are added
```

Comment:
For now, we use `numMethods` to describe the size of `{moveAccountBalance, createAccount}`, generate a random `int` to `switch` off of, and choose which method to run. This whole `run` method can be implemented in various ways, but I found this to be simple enough.